tools/aiecc/aiecc/main.py

```diff
@@ -173,10 +173,7 @@ async def process_arm_cgen(self):
 
       # Generate the included host interface
       file_physical = os.path.join(self.tmpdirname, 'input_physical.mlir')
-      if(self.opts.pathfinder):
-        await self.do_call(task, ['aie-opt', '--aie-create-pathfinder-flows', '--aie-lower-broadcast-packet', '--aie-create-packet-flows', '--aie-lower-multicast', self.file_with_addresses, '-o', file_physical]);
-      else:
-        await self.do_call(task, ['aie-opt', '--aie-create-flows', '--aie-lower-broadcast-packet', '--aie-create-packet-flows', '--aie-lower-multicast', self.file_with_addresses, '-o', file_physical]);
+      await self.do_call(task, ['aie-opt', '--aie-create-pathfinder-flows', '--aie-lower-broadcast-packet', '--aie-create-packet-flows', '--aie-lower-multicast', self.file_with_addresses, '-o', file_physical]);
```

Comment:
If you remove the option to use the basic router, the --pathfinder command line option should be removed from cl_arguments.py, and the files implementing the basic router. Alternatively, you could revert this and change the default for the --pathfinder cl_argument to be True and add a --no-pathfinder option. My vote is for the first option.

Reply:
Set pathfinder as default router, removed --pathfinder option and removed source files for old router.